src/commons.rs

```diff
@@ -554,3 +556,66 @@ pub extern "C" fn z_str_null() -> z_owned_str_t {
 pub extern "C" fn z_str_loan(s: &z_owned_str_t) -> *const libc::c_char {
     s._cstr
 }
+
+#[repr(C)]
+#[derive(Clone, Copy, Debug)]
+pub enum zc_locality_t {
```

Comment:
`Locality` is unstable in Rust because the full semantics has not been defined yet. 
While in Rust you need to explicitly enable the `unstable` feature, in C it is more cumbersome.
One way to signal the user that what is using is unstable is via a naming convention.

I propose the following naming rule for everything unstable: `zcu_xxx`. Once the API is eventually stabilized, we can keep compatibility with existing code using unstable marking by adding a `#define zc_xxx zcu_xxx`.

@p-avital @milyin opinions?

Reply:
Approved, we just discussed this in DM

Reply:
Why can’t this also be limited by "#define"s, or is it difficult from the point of view of automatic code generation?

Reply:
I believe the main reason is about the binary library we deliver for `zenoh-c`. A `#define` on the actual code compilation would create a combinatory problem for packaging.